packages/ui/components/pagination/Pagination.tsx

```diff
@@ -12,7 +12,6 @@ export interface PaginationProps {
   currentPage: number;
   pageSize: number;
   totalItems: number;
-  onPageChange: (page: number) => void;
   onPageSizeChange: (pageSize: number) => void;
```

Comment:
I wouldn't change this, as it's a very base component in our UI package that can be used in other places (in the future).